features/step_definitions/metadata.js

```diff
@@ -81,9 +74,9 @@ module.exports = function server() {
     );
 
     this.Then(/^a record of the metadata is stored$/, { timeout: TIMEOUT }, () => {
-        Assert.ok('foo', this.meta);
-        Assert.ok('bar', this.meta);
-        Assert.equal('foobar', this.meta.foo);
-        Assert.equal('barbaz', this.meta.bar);
+        Object.keys(this.expectedMeta).forEach((key) => {
+            Assert.ok(key, this.meta);
```

Comment:
Right, got it. Even without using`Assert.property`, I checked that I was expecting the test. Since this assert is not necessary, I deleted it.